lib/claim_documents/monitor.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'logging/monitor'
+
+module ClaimDocuments
+  ##
+  # Monitor functions for Rails logging and StatsD
+  # @todo abstract, split logging for controller and sidekiq
+  #
+  class Monitor < ::Logging::Monitor
+    # statsd key for document uploads
+    DOCUMENT_STATS_KEY = 'api.document_upload'
```

Comment:
`api.claim_documents` instead?

Reply:
@wayne-weibel sure, i can rename it. But we'll also need to remember to update the key name on the widget @balexandr created in Datadog as well.